enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -79,23 +82,30 @@ export function useLoadQuestion({
   // Avoid re-running the query if the parameters haven't changed.
   const sqlParameterKey = getParameterDependencyKey(initialSqlParameters);
 
-  const [loadQuestionState, loadQuestion] = useAsyncFn(async () => {
-    const state = await dispatch(
-      runQuestionOnLoadSdk({
+  const [loadQuestionState, loadAndQueryQuestion] = useAsyncFn(async () => {
+    const questionState = await dispatch(
+      loadQuestionSdk({
         options,
         deserializedCard,
         cardId,
-        cancelDeferred: deferred(),
```

Comment:
I wonder why the deferred for request cancellation `cancelDeferred: deferred()` was removed? afaik this allows other subsequent requests to cancel the existing requests. I noticed the below `runQuestionQuerySdk` also have `cancelDeferred` intact.

Reply:
this cancelDeferred was passed to `runQuestionQuerySdk` which was inside `runQuestionOnLoadSdk` (pre-change), since we extract that part out and put it after this `loadQuestionSdk` call (after-change) we don't need this `cancelDeferred` anymore.